workspaces/host/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+cargo-features = ["resolver"]
+
+[workspace]
+members = [
+
+  # Data generators
+  "crates/sdk/data-generators/csv-encoder",
+  "crates/sdk/data-generators/idash2017-logistic-regression-generator",
+  "crates/sdk/data-generators/image-processing-generator",
+  "crates/sdk/data-generators/intersection-set-sum-generator",
+  "crates/sdk/data-generators/linear-regression-generator",
+  "crates/sdk/data-generators/moving-average-convergence-divergence-generator",
+  "crates/sdk/data-generators/number-stream-generator",
+  "crates/sdk/data-generators/private-set-intersection-generator",
+  "crates/sdk/data-generators/private-set-intersection-sum-generator",
+  "crates/sdk/data-generators/shamir-secret-sharing",
+  "crates/sdk/data-generators/string-generator",
+  "crates/sdk/freestanding-execution-engine",
+  "crates/test-collateral/generate-policy",
```

Comment:
With the name `host` I would have expected `veracruz-server` to be in here.

Reply:
`host` is where the Veracruz is being built and agnostic of the platform. `<x>-host` is the workspace for the untrusted  part of the target platform is built (where `<x>` is currently `linux`, `nitro` and `icecap`).

I realize this is confusing. I would be happy if we could find a better terminology.

Reply:
Perhaps `common-host`?

Reply:
`common-host` sounds good to me.

Reply:
I don't think `common-host` clears up the confusion. The issue is that for `host` host refers to the cross-compiler terminology, whereas for `<x>-host` host refers to a computer. We need to remove "host" in one case:
- either for `host`: `local`, `build`.
- or for `<x>-host`: `<x>-target`, `<x>-server`, `<x>-untrusted`, etc.